File: devtools/project-core-extension-codestarts/src/main/resources/codestarts/quarkus/extension-codestarts/rest-codestart/java/src/test/java/org/acme/{resource.class-name}Test.tpl.qute.java
package org.acme;

import io.quarkus.test.junit.QuarkusTest;
import org.junit.jupiter.api.Test;

import static io.restassured.RestAssured.given;
import static org.hamcrest.CoreMatchers.is;

@QuarkusTest
class {resource.class-name}Test {
    @Test
    void testHelloEndpoint() {
        given()
          .when().get("{resource.path}")
          .then()
             .statusCode(200)
             .body(is("{resource.response}"));
    }

}

File: devtools/project-core-extension-codestarts/src/main/resources/codestarts/quarkus/core/examples/resteasy-reactive-example/kotlin/src/test/kotlin/org/acme/{resource.class-name}Test.tpl.qute.kt
package org.acme

import io.quarkus.test.junit.QuarkusTest
import io.restassured.RestAssured.given
import org.hamcrest.CoreMatchers.`is`
import org.junit.jupiter.api.Test

@QuarkusTest
class {resource.class-name}Test {

    @Test
    fun testHelloEndpoint() {
        given()
          .`when`().get("{resource.path}")
          .then()
             .statusCode(200)
             .body(`is`("{resource.response}"))
    }

}